src/index.js

```diff
@@ -0,0 +1,125 @@
+
+/*import './index.css';*/
+
+const taskinput = document.querySelector('.insert-text');
+const content = document.querySelector('.content');
+// function
+let editId;
+let isEditedTask = false;
+// JSON.parse(localStorage.getItem("todo-list"))
+let todos;
+if (!localStorage.getItem('todo-list')) {
+  todos = [];
+} else {
+  todos = JSON.parse(localStorage.getItem('todo-list'));
+}
+function showTodo() {
+  let li = ' ';
+  if (todos) {
+    todos.forEach((todo, id) => {
+      li += ` <div id="task">   
+            <input  onclick="updateStatus(this)"  type="checkbox" id="${id}"  >
+               <p>  ${todo.description} </p>     
+             <div class="setting">
+           <i  onclick="showMenu(this)"  class="fa fa-ellipsis-v menu" aria-hidden="true"></i>
+        <ul class="task-menu">
+         <li class="pepe"  onclick="editTask(${id}, '${todo.description} ') "> <i class="fa fa-pencil" aria-hidden="true"></i>edit </li>  
+        <li  onclick="deleteTask(${id})"> <i class="fa fa-trash" aria-hidden="true"></i>remove </li>  
+       </ul>
+      </div>
+    </div>`;
+    });
+  }
+  content.innerHTML = li;
+}
+showTodo();
+
+function updateStatus(selectedTask) {
+  /* console.log(selectedTask) */
+  // getting paragraph that contains task name
+  const taskName = selectedTask.parentElement.childNodes[3]; // esto deberia subrayar el texto
+  if (selectedTask.checked) {
+    taskName.classList.add("checked");
+    // optional for the delted button completed
+    todos[selectedTask.id].completed = "completed";
```

Comment:
- [ ] [OPTIONAL] Great implementation @latinogan , however, I will recommend we change the "completed" to "true", this will make it a boolean `completed`: `true`/`false`.